### PR TITLE
bug: title in a centered key value layout is left aligned

### LINKED ISSUE #3036: [BUG] Title in a centered key value layout is left-aligned
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
PR #3027 has introduced a layout bug, that renders the title in a centered key value scenario as left aligned on small screens and in the first column of the 3-column layout on large screens (see screenshots below 👇🏻 )  

## Describe how to reproduce the bug

1. Go to https://cookbook.kirby.design/#/home/showcase/header
2. Scroll to "Combined" example
3. See error as described above

## Which Kirby version was used?
8.4.1

## What was the expected behavior?
Title should be centered.

## Add any screenshots
Small screen:
image

Large screen:
image

## Please complete the following information:
- OS: [e.g. Windows, MacOS, iOS]
- Browser [e.g. Chrome, Safari]
- Browser version [e.g. 22]

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [x] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [x] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [x] Signal to others you are working on the issue by assigning yourself.
- [x] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [x] Create a pull-request.

### PR DESCRIPTION
**Pull request by jakobe**
## Which issue does this PR close?

This PR closes #3036 

## What is the new behavior?

Centers title as expected.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: apps/cookbook/src/app/examples/header-example/examples/combined.ts [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import { Component } from '@angular/core';
+import { ToastConfig, ToastController } from '@kirbydesign/designsystem';
 
 import { config as customSectionConfig } from './custom-section';
 
 const config = {
   selector: 'cookbook-header-example-combined',
-  template: `<kirby-header [title]="'Title'" value="12.345,67" valueUnit="USD" subtitle1="Subtitle one" subtitle2="Subtitle two">
+  template: `<kirby-header [title]="'Title'" value="12.345,67" valueUnit="USD" subtitle1="Subtitle one" subtitle2="Subtitle two" (titleClick)="onTitleClick($event)">
+  <kirby-icon name="arrow-down" *kirbyHeaderTitleActionIcon></kirby-icon>
```
Comment (RasmusKjeldgaard) [inline apps/cookbook/src/app/examples/header-example/examples/combined.ts:9]: For some reason I am not seeing this reflected in the combined example in the cookbook. There is no clickable title or arrow icon 🤔 Or am I looking at the wrong thing? 

image

**Review by RasmusKjeldgaard — CHANGES_REQUESTED** [review]
CSS looks great, so just one question about the cookbook example 👍

File: apps/cookbook/src/app/examples/header-example/examples/combined.ts (hunk 1 above)
Comment (RasmusKjeldgaard) [reply apps/cookbook/src/app/examples/header-example/examples/combined.ts:9]: Ah wait, this branch probably just needs to be synced with develop for this to make sense.

Comment (RasmusKjeldgaard) [reply apps/cookbook/src/app/examples/header-example/examples/combined.ts:9]: Yep, that was it.

Now that I see it, the toast message seems a little disconnected from the example container, consider wrapping it in the iphone/iframe component to circumvent that.

Comment (jakobe) [reply apps/cookbook/src/app/examples/header-example/examples/combined.ts:9]: @RasmusKjeldgaard I see your point, but we only wrap examples in the iPhone frame, when the example is related to Page, i.e. we need to see how it scrolls, fades into the toolbar etc. This example doesn't showcase that.
And on another note, it's not different from the Toast example itself, that isn't wrapped either: https://cookbook.kirby.design/#/home/showcase/toast

**Review by RasmusKjeldgaard — APPROVED** [review]

**Pull request merged**